Standards/scs-0001-v2-sovereign-cloud-standards.md

```diff
@@ -0,0 +1,309 @@
+| `type`          | REQUIRED                                                                   | one of `Procedural`, `Standard`, or `Decision Record`                                 |
+| `status`        | REQUIRED                                                                   | one of `Proposal`, `Draft`, `Stable`, `Deprecated`, or `Rejected`                     |
+| `track`         | REQUIRED                                                                   | one of `Global`, `IaaS`, `KaaS`, `IAM`, `Ops`                                         |
+| `obsoleted_at`  | REQUIRED if `status` is `Deprecated`                                       | ISO formatted date indicating the date after which the deprecation is in effect       |
+| `stabilized_at` | REQUIRED if `status` was ever `Stable`                                     | ISO formatted date indicating the date after which the document was considered stable |
+| `rejected_at`   | REQUIRED if `status` is `Rejected`                                         | ISO formatted date indicating the date on which the document was rejected             |
+| `replaced_by`   | RECOMMENDED if `status` is `Deprecated` or `Rejected`, FORBIDDEN otherwise | List of documents which replace this document.                                        |
+| `replaces`      | OPTIONAL                                                                   | List of documents which this document replaces.                                       |
+
+### Sections
+
+#### Standard
+
+Each Standard document SHOULD have the following sections:
+
+- An _Introduction_ providing context on the document and linking to other relevant materials.
+- A _Terminology_ section which shortly describes terms used in the document, including possible abbreviations.
+- A _Motivation_ section which details why this document or the thing it describes is necessary.
+- A _Standard_ section containing the actual standardization decision.
+
+In addition, the following OPTIONAL sections should be considered:
+
+- A _Design Considerations_ section for Standard type documents,
+  which details other choices
+  which have been considered for the specific feature
+  but were ultimately rejected.
+- An _Open Questions_ section which links to issues
+  detailing any open discussion points with respect to a document.
+  This section is RECOMMENDED during the discussion phase (pre 1.0.0)
+  as a "table of contents" of things to work on in that context.
+- A _Related Documents_ section which references related Standards
+  or Decisions, both upstream and/or other SCS documents.
+- A _Conformance Tests_ section that contains hints on how to validate
+  conformance with this spec, ideally links to conformance test cases.
+
+#### Decision Record
+
+Each Decision Record document SHOULD have the following sections:
```

Comment:
Changed it. But I just want to say, that I had the same confusion about "SHOULD" and "MUST" again (like I always have) and the previous creator also wrote this standard (in the `Standard` section) with a "SHOULD".
We could summon him and ask why he decided on that, but I doubt he will remember after 1,5years and TBF I think the change to "MUST" is good.

Reply:
> Changed it. But I just want to say, that I had the same confusion about "SHOULD" and "MUST" again (like I always have) and the previous creator also wrote this standard (in the `Standard` section) with a "SHOULD". We could summon him and ask why he decided on that, but I doubt he will remember after 1,5years and TBF I think the change to "MUST" is good.

I don't understand this, if you reference concrete stuff from elsewhere please cite it or put a link to it. Because I can't find what you mean in the "Standard" section of this document.

Do you mean the "Sections" section instead? It looks likely you did.
I didn't know this was inspired by this section, still reasoning for keeping things optional should probably be documented - as a decision record, pun intended - when altering the standard with respect to decision records.